repository/src/main/java/org/apache/atlas/repository/store/graph/v2/tasks/ClassificationPropagationTasks.java

```diff
@@ -28,32 +28,36 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.List;
 import java.util.Map;
 
 public class ClassificationPropagationTasks {
-    private static final Logger LOG      = LoggerFactory.getLogger(ClassificationPropagationTasks.class);
+    private static final Logger LOG = LoggerFactory.getLogger(ClassificationPropagationTasks.class);
+
     public static class Add extends ClassificationTask {
         public Add(AtlasTask task, AtlasGraph graph, EntityGraphMapper entityGraphMapper, DeleteHandlerDelegate deleteDelegate, AtlasRelationshipStore relationshipStore) {
             super(task, graph, entityGraphMapper, deleteDelegate, relationshipStore);
         }
 
         @Override
-        protected void run(Map<String, Object> parameters) throws AtlasBaseException {
-            String entityGuid               = (String) parameters.get(PARAM_ENTITY_GUID);
-            String toEntityGuid               = (String) parameters.get(PARAM_TO_ENTITY_GUID);
-            String tagTypeName              = getTaskDef().getTagTypeName();
-            String parentEntityGuid         = getTaskDef().getParentEntityGuid();
+        protected void run(Map<String, Object> parameters, TaskContext context) throws AtlasBaseException {
+            String entityGuid = (String) parameters.get(PARAM_ENTITY_GUID);
+            String toEntityGuid = (String) parameters.get(PARAM_TO_ENTITY_GUID);
+            String tagTypeName = getTaskDef().getTagTypeName();
+            String parentEntityGuid = getTaskDef().getParentEntityGuid();
             Boolean previousRestrictPropagationThroughLineage = (Boolean) parameters.get(PARAM_PREVIOUS_CLASSIFICATION_RESTRICT_PROPAGATE_THROUGH_LINEAGE);
             Boolean previousRestrictPropagationThroughHierarchy = (Boolean) parameters.get(PARAM_PREVIOUS_CLASSIFICATION_RESTRICT_PROPAGATE_THROUGH_HIERARCHY);
 
             if (org.apache.atlas.service.FeatureFlagStore.isTagV2Enabled()) {
                 LOG.info("Using v2 tag flow (Cassandra) for Add propagation task");
-                entityGraphMapper.propagateClassificationV2(parameters, entityGuid, tagTypeName, parentEntityGuid, toEntityGuid);
+                List<String> impactedVertices = entityGraphMapper.propagateClassificationV2(parameters, entityGuid, tagTypeName, parentEntityGuid, toEntityGuid);
```

Comment:
can we return the int or long directly within propagateClassificationV2 instead returning a list to extract only size of it